### ISSUE TITLE
This crash is not in our database ( crash_id = 19548368, case_id = 230801355 )

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#13277]
> What application are you using?

Vision Zero Editor

> Describe the problem.

This crash is not in our database ( crash_id = 19548368, case_id = 230801355 )

> Is there anything else we should know?

This crash was available on May 22nd, 2023, but it is still not in our database. 

> Website Address

https://visionzero.austin.gov/editor/#/crashes/

Internet Browser: Edge

> Requested By
Xavier A.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/64d26c45fd1b500029b772a4/untitled.png) (35.0kb)

<img width="590" alt="Screen Shot 2023-09-28 at 1 51 52 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/5566727/e9dfebc3-ed17-4d5d-b112-6a0d1615335c">

Request ID: DTS23-109055

**Comment by patrickm02L** [comment]
## 10/3
Talked about this in VZ Sprint Review. 
We can do an export on crash date and do a backfill to import this record.

**Comment by frankhereford** [comment]
As we discussed, this was resolved by backfilling the database with an extract spanning from 3/20/2023 through 3/22/2023. I verified that the crash was in the extract and then ran it through the standard import process. Upon verifying that the records had successfully been created in the VZDB, I ran the CR3 downloader to get the PDF and then manually kicked off the OCR script to get this crash fully into the "normal record" state.

https://visionzero.austin.gov/editor/#/crashes/19548368
